content/user/advanced-modules/PDFTemplates.adoc

```diff
@@ -0,0 +1,116 @@
+---
+Title: PDF-Templates
+Weight: 85
+---
+
+
+:toc:
+:toc-title: Table of content
+
+:experimental:
+
+:imagesdir: ./../../../../images/en/user
+
+ifdef::env-github[:imagesdir: ../../../../../master/static/images/en/user/PDFTemplates]
```

Comment:
Also please remove all these `ifdef` blocks, they are only needed in the Russian files (see https://github.com/salesagility/SuiteDocs/issues/210)